src/main/resources/db/migration/V2024_05_15_1300__CIV-11832_hearing_documents_uploaded_scenario.sql

```diff
@@ -0,0 +1,30 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.CP.HearingDocuments.Claimant.Uploaded.Claimant',
+        '{Notice.AAA6.CP.HearingDocuments.Upload.Claimant}',
+        '{}'),
+       ('Scenario.AAA6.CP.HearingDocuments.Claimant.Uploaded.Defendant',
+        '{}',
+        '{}'),
+       ('Scenario.AAA6.CP.HearingDocuments.Defendant.Uploaded.Claimant',
+        '{}',
+        '{}'),
+       ('Scenario.AAA6.CP.HearingDocuments.Defendant.Uploaded.Defendant',
+        '{Notice.AAA6.CP.HearingDocuments.Upload.Defendant}',
+        '{}');
+
```

Comment:
I wonder if the naming is confusing - although I have no issues with the functionality, and will still approve.

Should we have it more like:
1. Scenario.AAA6.CP.HearingDocuments.Uploaded.Claimant -> When claimant has uploaded evidence at some point.
2. Scenario.AAA6.CP.HearingDocuments.NotUploaded.Claimant -> When claimant has uploaded nothing yet, but their dashboard needs updating.
3. Scenario.AAA6.CP.HearingDocuments.Uploaded.Defendant -> When defendant has uploaded evidence already, and update required.
4. Scenario.AAA6.CP.HearingDocuments.NotUploaded.Defendant -> When defendant has not uploaded yet, but dashboard update required.

I feel that change matches your actual implementation better (the current naming might be a holdover from the approach we initially discussed).

Reply:
Your suggestion sounds more readable. Changes done